api/src/test/java/io/kaoto/backend/api/resource/v2/IntegrationsResourceTest.java

```diff
@@ -418,14 +464,14 @@ void roundTrip(String file) throws IOException {
         var sourceCode = res2.extract().body().asString();
 
         //This should be the same as the original source code
```

Comment:
Sometimes, tests on my local machine failed here because the original yaml and generated yaml have different properties order `spec.source.properties`
```
    properties:
      apiKey: your own
      keywords: Apache Camel
      apiKeySecret: your own
      accessToken: your own
      accessTokenSecret: your own
```

```
properties:
      keywords: Apache Camel
      apiKey: your own
      apiKeySecret: your own
      accessToken: your own
      accessTokenSecret: your own
```

Comparing YAMLs as objects should resolve that problem.

Reply:
Do we have a good comparison tool for yaml objects? I couldn't find any, that's why I compare as strings. But compare as strings is a nightmare every time some minor change occurs (like change " by ')

Reply:
I don't know about any but I can try to find out some.